projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts

```diff
@@ -0,0 +1,13 @@
+import {tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Textarea`, () => {
+    [`m`, `l`].forEach(size => {
+        test(`size=${size}`, async ({page}) => {
+            await tuiGoto(page, `components/checkbox/API?size=${size}`);
+            const example = page.locator(`#demo-content`);
+
+            await expect(example).toHaveScreenshot(`checkbox-size-${size}.png`);
```

Comment:
Too wide screenshot

<img width="1166" alt="Screenshot 2023-11-20 at 13 37 01" src="https://github.com/taiga-family/taiga-ui/assets/35179038/2b697bc0-f826-42b6-8a0e-b3f377fa8a40">

Reply:
<img width="804" alt="Screenshot 2023-11-20 at 1 03 42 PM" src="https://github.com/taiga-family/taiga-ui/assets/29022394/2e48d718-c7e7-482a-b2d0-9a032dc6d46e">

The Cypress screenshot size was 1758x416, so I thought that was how wide we wanted the screenshot. Do you have any advice on what the size should be? I can try to force that size of the screenshot.

Reply:
You can use `await page.setViewportSize({width, height})` for this test.
For example, you can set it as `300*300`.

See the example:
https://github.com/taiga-family/taiga-ui/blob/1e624c0f36e68d02abc642b89e606402f57225cc/projects/demo-playwright/tests/core/breakpoint-service/breakpoint-service.spec.ts#L11

Reply:
Thank you, made this change